docs/user-guide/ze-profiles.md

```diff
@@ -120,3 +121,45 @@ You can use this example to customize your own profiles in a configuration file.
   "plugins": []
 }
 ```
+
+## Using base profiles and tokens with existing profiles
+
+As a Zowe user, you can leverage the base profile functionality to access multiple services through Single Sign-on. Base profiles enable you to authenticate using Zowe API Mediation Layer (API ML). You can use base profiles with more than one service profile. For more information, see [Base Profiles](../user-guide/cli-using-using-profiles-v1.md#base-profiles).
+
+Before you log in and connect your service profile, ensure that you have [Zowe CLI](../user-guide/cli-install-cli-checklist.md) v6.16 or higher installed.
+
+### Accessing services through API ML using SSO
+
+Connect your service profile with a base profile and token:
+
+1. Right-click on a profile on one of the ZE trees, the profiles you want to connect through with the api ml
+2. Select Manage Profile from the context menu
+3. In the quick pick, select Log in to Authentication Service
+4. In the next quick pick, select the appropriate option for authenticating to the API ML (user/pw or a cert file)
+5. Enter the into prompted for (credentials, or browsing to the certificate file)
```

Comment:
"into" should be "info". Something is weird about the wording/verb tense in this line.